src/net.rs

```diff
@@ -47,8 +76,8 @@ const PROTOCOL_VERSION : u8 = 1;
 /// is established.
 #[derive(RustcEncodable, RustcDecodable)]
 pub struct Greeting {
-    protocol_version: u8,   // 1 byte
-    message: String,        // n bytes
+    protocol_version : u8,   // 1 byte
+    message : String,        // n bytes
```

Comment:
On the left side of a colon there is no whitespace... at least in this context :P